libpamc/Makefile.am

```diff
@@ -22,3 +22,7 @@ endif
 lib_LTLIBRARIES = libpamc.la
 
 libpamc_la_SOURCES = pamc_client.c pamc_converse.c pamc_load.c
+
+# Pkg-config script.
+pkgconfigdir = $(libdir)/pkgconfig
+pkgconfig_DATA = pam.pc
```

Comment:
This has to be `pamc.pc`, otherwise the build fails with the following diagnostics:
`make[4]: *** No rule to make target 'pam.pc', needed by 'all-am'.`

Reply:
Done, I thought I had changed that when I copied these lines from libpam/Makefile.am.

I built the project on my machine (not sure why I didn't take the time to do so before now) and everything seems to build correctly. (And make check doesn't show any problem)